components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -327,4 +328,37 @@ public static boolean useOrganizationRolesForValidation(String organizationId) {
         }
         return true;
     }
+
+    /**
+     * Return whether tenant qualified paths are supported for first level organizations in the deployment.
+     *
+     * @return True if tenant qualified paths are supported for first level organizations.
+     */
+    public static boolean isTenantQualifiedPathsSupportedForLevelOneOrganizations() {
```

Comment:
```suggestion
    public static boolean isOnlyTenantQualifiedPathsSupportedForLevelOneOrganizations() {
```

or else it would better if `isOrganizationQualifiedPathsDisabledForLevelOneOrganizations` and update the method descriptions accordingly

Reply:
Addressed with 7f0358f34132558c2384544685555ae6c399c810